scripts/deploy_gke.sh

```diff
@@ -58,8 +58,19 @@ if [[ $ENV == "autopush" ]]; then
   # Update bigtable and bigquery version
   gsutil cp gs://datcom-control/latest_base_cache_version.txt deploy/storage/bigtable.version
   gsutil cp gs://datcom-control/latest_base_bigquery_version.txt deploy/storage/bigquery.version
+  # Import group
+  # TODO: Loop through gs://datcom-control/autopush/datcom-control/autopush
+  > deploy/storage/bigtable_import_groups.version
+  for group in frequent infrequent ipcc
```

Comment:
why not:

```
for src in $(gsutil ls gs://datcom-control/autopush/*_latest_base_cache_version.txt); do
  echo "Copying $src"
  gsutil cat $src >> deploy/storage/bigtable_import_groups.version
done
```

Reply:
we could also require the files have a newline at the end

Reply:
This is much better bash! Also since echo produces a newline, we should be fine here even. the original txt has no newline.